examples/realm-java-compatibility/app/src/main/java/io/realm/kotlin/demo/javacompatibility/data/java/JavaRepository.kt

```diff
@@ -45,12 +45,12 @@ class JavaRepository(appContext: Context) : Repository {
 
     init {
         Realm.init(appContext)
-        realm = Realm.getInstance(
-            RealmConfiguration.Builder()
-                .name("java.realm")
-                .allowWritesOnUiThread(true)
-                .build()
-        )
+        val config = RealmConfiguration.Builder()
+            .name("java.realm")
+            .allowWritesOnUiThread(true)
+            .build()
+        Realm.deleteRealm(config)
```

Comment:
Nit pick. Maybe a bit aggressive, maybe we could just `deleteIfMigrationRequired` ... or what it is called.

Reply:
I added this because we are using `copyToRealm` in the Repository...something was causing the Realm to not be cleaned up which caused every subsequent test to fail with "Primary key already exists".

So while it is a work-around, it is similar to our tests using a random test directory. Using `deleteIfMigrationRequired` wouldn't help in this case.

Reply:
Ok, let's just wipe it 🧹